sdk/core/core/inc/az_precondition_failed.h

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+#ifndef _az_PRECONDITION_INTERNAL_H
```

Comment:
I prefer the header be az_precondition.h.  The ifndef should match the header name as well.

Reply:
my bad.  thank you

Reply:
Done